workers/unity/Packages/com.improbable.gdk.buildsystem/Util/BuildSupportChecker.cs

```diff
@@ -0,0 +1,79 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using UnityEditor;
+using UnityEngine;
+
+public static class BuildSupportChecker
+{
+    private static readonly Dictionary<BuildTarget, string> BuildPlatformSupportModuleDirectoryNames;
+
+    static BuildSupportChecker()
+    {
+        BuildPlatformSupportModuleDirectoryNames = new Dictionary<BuildTarget, string>();
+
+        const string linuxStandaloneSupport = "LinuxStandaloneSupport";
+
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinux] = linuxStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinux64] = linuxStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneLinuxUniversal] = linuxStandaloneSupport;
+
+#if !UNITY_EDITOR_OSX
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneOSX] = "MacStandaloneSupport";
+#endif
+
+#if !UNITY_EDITOR_WIN
+        var windowsStandaloneSupport = "WindowsStandaloneSupport";
+
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneWindows] = windowsStandaloneSupport;
+        BuildPlatformSupportModuleDirectoryNames[BuildTarget.StandaloneWindows64] = windowsStandaloneSupport;
+#endif
+    }
+
+    public class BuildSupportCheckResult
+    {
+        public readonly bool CanBuild;
+        public readonly BuildTarget[] TargetsWithoutBuildSupport;
+
+        public BuildSupportCheckResult(bool canBuild, BuildTarget[] targetsWithoutBuildSupport)
+        {
+            CanBuild = canBuild;
+            TargetsWithoutBuildSupport = targetsWithoutBuildSupport;
+        }
+    }
+
+    public static BuildSupportCheckResult CheckBuildSupport(params BuildTarget[] buildTargets)
+    {
+        string editorExePath = Environment.GetCommandLineArgs()[0];
```

Comment:
I think there's a path in `Application` or `EditorApplication` that may point to this directory?
`args[0]` may not always be reliable

Reply:
Aye, `EditorApplication.applicationPath` :)